src/infi/systray/win32_adapter.py

```diff
@@ -80,11 +82,25 @@
     LRESULT = ctypes.c_longlong
 HANDLE = ctypes.c_void_p
 
-def convert_to_ascii(s):
+# On Win 2000 and later, tray hover text can be 128 characters, otherwise 64
+# https://msdn.microsoft.com/en-us/library/windows/desktop/bb773352%28v=vs.85%29.aspx
+# https://msdn.microsoft.com/en-us/library/ms724834%28v=vs.85%29.aspx
+if sys.getwindowsversion().major >= 5:
+    SZTIP_MAX_LENGTH = 128 
+else:
+    SZTIP_MAX_LENGTH = 64

```

Comment:
In any case I'm pretty sure we can drop pre-2000 support - let's change the size to 128 anyway.